MH4U Dex/RegionEncyclopediaCollectionViewController.m

```diff
@@ -56,10 +59,27 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cell
     RegionEncylcopediaCollectionViewCell *cell = [collectionView dequeueReusableCellWithReuseIdentifier:NSStringFromClass([RegionEncylcopediaCollectionViewCell class]) forIndexPath:indexPath];
     
     // Configure the cell
-    cell.backgroundColor = [UIColor blueColor];
+    Region *region = self.regions[indexPath.row];
+    cell.regionName = region.name;
+    cell.regionKeyName = region.keyName;
+    [cell displayContents];
     return cell;
 }
 
-#pragma mark <UICollectionViewDelegate>
+#pragma mark - Navigation
+
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
+{
+    if ([segue.identifier isEqualToString:@"showRegionDetails"]) {
+        RegionContainerViewController *regionVC = (RegionContainerViewController *)segue.destinationViewController;
+        regionVC.managedObjectContext = self.managedObjectContext;
+        regionVC.regionName = [sender regionName];
+        regionVC.regionKeyName = [sender regionKeyName];
+        
+        CoreDataController *coreDataController = [CoreDataController sharedCDController];
+        NSPredicate *predicate = [NSPredicate predicateWithFormat:@"%K == %@", @"name", [sender regionName]];
```

Comment:
Does mogenerator provide you a constant you can use in place of the `@"name"` string literal?